agent/lib/kontena/network_adapters/weave.rb

```diff
@@ -201,6 +201,7 @@ def start(info)
         end
       end
 
+      attach_router unless interface_ip('weave')
```

Comment:
Like this:

```
$ ip addr show weave
26: weave: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1410 qdisc noqueue state UP group default qlen 1000
    link/ether c6:3f:e0:48:25:8d brd ff:ff:ff:ff:ff:ff
    inet6 fe80::c43f:e0ff:fe48:258d/64 scope link 
       valid_lft forever preferred_lft forever
```

Giving:

```
kontena-agent | E, [2016-10-31T07:29:52.586188 #1] ERROR -- Kontena::NetworkAdapters::Weave: Errno::EADDRNOTAVAIL: Address not available
kontena-agent | D, [2016-10-31T07:29:52.586289 #1] DEBUG -- NetworkAdapters::Weave: /app/lib/kontena/helpers/iface_helper.rb:21:in `ioctl'
kontena-agent | /app/lib/kontena/helpers/iface_helper.rb:21:in `interface_ip'
kontena-agent | /app/lib/kontena/helpers/iface_helper.rb:32:in `interface_ip'
kontena-agent | /app/lib/kontena/network_adapters/weave.rb:204:in `start'
kontena-agent | /usr/lib/ruby/gems/2.3.0/gems/celluloid-0.17.3/lib/celluloid/calls.rb:28:in `public_send'
kontena-agent | /usr/lib/ruby/gems/2.3.0/gems/celluloid-0.17.3/lib/celluloid/calls.rb:28:in `dispatch'
kontena-agent | /usr/lib/ruby/gems/2.3.0/gems/celluloid-0.17.3/lib/celluloid/call/async.rb:7:in `dispatch'
kontena-agent | /usr/lib/ruby/gems/2.3.0/gems/celluloid-0.17.3/lib/celluloid/cell.rb:50:in `block in dispatch'
kontena-agent | /usr/lib/ruby/gems/2.3.0/gems/celluloid-0.17.3/lib/celluloid/cell.rb:76:in `block in task'
kontena-agent | /usr/lib/ruby/gems/2.3.0/gems/celluloid-0.17.3/lib/celluloid/actor.rb:339:in `block in task'
kontena-agent | /usr/lib/ruby/gems/2.3.0/gems/celluloid-0.17.3/lib/celluloid/task.rb:44:in `block in initialize'
kontena-agent | /usr/lib/ruby/gems/2.3.0/gems/celluloid-0.17.3/lib/celluloid/task/fibered.rb:14:in `block in create'
```

That doesn't happen normally, but if something fails between the `launch-router`/`attach-router` creating the bridge, and then the `expose` configuring it.